helsingborg-stad/aterbruket-webapp
# Feature/detail page styling

**Explain the changes you’ve made**
Styled the detail page according to design in Figma
Also changed the way the header behaves, and what buttons to show depending on the items status

**Explain why these changes are made**
To get a nicer and more "done" look

**Explain your solution**
Structured all the info in details page after the design and worked with classNames and styled components to get the styling
Also a lot of conditionals depending on the items status

**How to test the changes?**
Click on an advert and have a look at the detail page, check both available and reserved items to see different buttons and headers
NOTE: "ta bort reservation" - function is not in place (another task for this)
NOTE: button in header if scrolled past the big button - function is not in place (another task for this)
